### PR TITLE
Help menu new

### PR DESCRIPTION
**Pull request by papaiza**
Implemented the help menu/button. Button opens the user manual. Help menu contains an explicit link to the user manual. Sample input tabs included in help menu.

### CODE REVIEW

File: src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/SampleInput_1Listener.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    @Override
+    public void actionPerformed(ActionEvent e) {
+    	open(createURI(INPUT_FILE_1));
+    	
+    }
+    /**
+	 * Opens the URI in the URI object using the desktop's default browser
+	 * @param URI uri
+	 */
+	public static void open(URI uri){
```
Comment (MarkyC) [inline src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/SampleInput_1Listener.java]: [DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is violated here, perhaps open/createURI should be in some utility class?

File: src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/HelpListener.java [hunk 2]
```diff
@@ -30,13 +34,35 @@ public HelpListener(MainJFrame window) {
...
-        helpFrame.add(label);
-        helpFrame.setVisible(true);
+    	open(createURI(USER_MANUAL_LINK));
+    	
+    }
+    /**
+	 * Opens the URI in the URI object using the desktop's default browser
+	 * @param URI uri
+	 */
+	public static void open(URI uri){
```
Comment (MarkyC) [inline src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/HelpListener.java]: [DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is violated here, perhaps open/createURI should be in some utility class?

File: src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/SampleInput_2Listener.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    @Override
+    public void actionPerformed(ActionEvent e) {
+    	open(createURI(INPUT_FILE_2));
+    	
+    }
+    /**
+	 * Opens the URI in the URI object using the desktop's default browser
+	 * @param URI uri
+	 */
+	public static void open(URI uri){
```
Comment (MarkyC) [inline src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/SampleInput_2Listener.java]: [DRY](http://en.wikipedia.org/wiki/Don%27t_repeat_yourself) is violated here, perhaps open/createURI should be in some utility class?

File: src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/HelpListener.java (hunk 2 above)
Comment (papaiza) [inline src/main/java/ca/yorku/cse2311/tab2pdf/ui/listener/HelpListener.java]: Will fix tonight.

**Comment by MarkyC** [comment]
Good enough. I would have liked to see something like:

```
public class URLHelper {
    public static void open(String link) throws IOException, URISyntaxExecption;
}
```

so it can be called in code like `URLHelper.open("http://google.com");`, but we can refactor that later.

### *Why is doing a lot in constructors bad?*

* It makes it hard to test
* back in *ye olden days*, throwing Exceptions in constructors could potentially not be caught and handled; also of note was that it was possible for constructors to fail (not sure if this is true in Java)
* It violates the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle). If you wanted to later add another method to OpenUrl, it would be very difficult, since calling the constructor already does some task

**Pull request merged**